app/src/main/java/com/example/test/automationtest/util/HogeUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.example.test.automationtest.util;
+
+public class HogeUtil {
+
+    public String hoge(int arg){
+        String reutrnValeu="";
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/4hin/android-automation-test/blob/ebc5a11718c7e13caf5ea118d562632b988d5943/app/src/main/java/com/example/test/automationtest/util/HogeUtil.java#L6"></span>WhitespaceAround: ‘=’ is not preceded with whitespace.</td>
    </tr>
  </tbody>
</table>